test/e2e.method.signing.js

```diff
@@ -394,22 +394,26 @@ describe('transaction and message signing [ @E2E ]', function() {
     // Smoke test to validate browserify's buffer polyfills (feross/buffer@5)
     // A companion regression test for Webpack & feross/buffer@4.9.2 exists at:
     // test/eth.accounts.webpack.js
-    it("encrypt then decrypt wallet", async function() {
-        this.timeout(10000);
-
-        const password = "qwerty";
-        const addressFromWallet = wallet[0].address;
+    it("encrypt then decrypt wallet", function(done) {
```

Comment:
the test was changed to be async, you need to make sure that `done()` is called resolve the test

Reply:
done is called on line 413
it possible the timeout is not long enough :{
but hopefully the rerun will fix that.. something in the new update of firefox changed execution time of encrypt